XpringKit/XRP/XRPClientProtocol.swift

```diff
@@ -0,0 +1,47 @@
+/// An interface into the XRP Platform.
+public protocol XRPClientProtocol {
```

Comment:
So just to put this into TypeScript terms (which I understand), this defines an `interface` with basically `abstract` methods that both `XRPClient` and `FakeXRPClient` implement.

When you later test the `XpringClient`, you can create a class/instance that implements this interface, but returns known values so you can pass your test assertions, correct?

Reply:
Yup. `protocol` in swift == `interface` in typescript. Building this here lets us fake this later.